components/skeleton/skeleton-mixin.js

```diff
@@ -0,0 +1,101 @@
+import '../colors/colors.js';
+import { css } from 'lit-element/lit-element.js';
+import { dedupeMixin } from '@open-wc/dedupe-mixin';
+
+export const skeletonStyles = css`
+	@keyframes loadingPulse {
+		0% { background-color: var(--d2l-color-sylvite); }
+		50% { background-color: var(--d2l-color-regolith); }
+		75% { background-color: var(--d2l-color-sylvite); }
+		100% { background-color: var(--d2l-color-sylvite); }
+	}
+	:host([skeleton]) .d2l-skeletize::before {
+		animation: loadingPulse 1.8s linear infinite;
+		background-color: var(--d2l-color-sylvite);
+		border-radius: 0.2rem;
+		bottom: 0;
+		content: '';
+		left: 0;
+		position: absolute;
+		right: 0;
+		top: 0;
+		z-index: 2000;
+	}
+	:host([skeleton]) .d2l-skeletize {
+		border-color: transparent;
```

Comment:
@mdgbayly The one change I made to your original styles was to use a transparent border instead of `border: none`. Otherwise elements that had a border would change heights as they went from skeleton to not.